node/src/types/json_compatibility/account.rs

```diff
@@ -64,6 +68,21 @@ pub struct Account {
     action_thresholds: ActionThresholds,
 }
 
+impl Debug for Account {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("Account")
+            .field("account_hash", &self.account_hash)
+            .field(
+                "named_keys",
+                &LimitSlice::<_, 10>::new(self.named_keys.as_slice()),
+            )
+            .field("main_purse", &self.main_purse)
+            .field("associated_keys", &self.associated_keys)
```

Comment:
This can include up to 100 entries as well.

Reply:
It's not unbounded though, is it? Is it possible that 100 entries is still "reasonable"?

Reply:
It's a `BTreeMap<AccountHash, Weight>`.

Reply:
So that's around 5 KB-ish? Not great, but not as as bad I think.